deeppavlov/models/doc_retrieval/bpr.py

```diff
@@ -0,0 +1,101 @@
+from typing import Any, List, Tuple
+
+import faiss
+import numpy as np
+import torch
+from tqdm import trange
+from transformers import AutoTokenizer, BertModel
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+from deeppavlov.core.models.serializable import Serializable
+
+
+@register('concat_lists')
+def concat_lists(list_a: List[List[Any]], list_b: List[List[Any]]):
+    list_u = []
+    for element_a, element_b in zip(list_a, list_b):
+        list_u.append(element_a + element_b)
+    return list_u
```

Comment:
move to separated module. If you keep it here, you will need to add concat_lists to requirements registry